src/test/java/cms/gov/madie/measure/resources/MeasureControllerMvcTest.java

```diff
@@ -349,6 +342,36 @@ public void testNewMeasureNameMustNotBeNull() throws Exception {
     verifyNoInteractions(measureRepository);
   }
 
+  @Test
+  public void testCreateCmsId() throws Exception {
+    final MeasureSet measureSet =
+        MeasureSet.builder()
+            .id("f225481c-921e-4015-9e14-e5046bfac9ff")
+            .cmsId(6)
+            .measureSetId("measureSetId")
+            .owner("test.com")
+            .acls(null)
+            .build();
+
+    when(measureSetService.createCmsId(anyString(), anyString(), anyString()))
+        .thenReturn(measureSet);
+
+    final String measureSetAsJson = toJsonString(measureSet);
+    mockMvc
```

Comment:
there is one codacy violation. it expects test case to have some assertion/verification

Reply:
thanks, I just fixed it